ui/pages/liquidity.tsx

```diff
@@ -337,8 +337,23 @@ export default function Liquidity() {
                     <ActionButton
                       className="btn btn-primary normal-case font-medium w-full"
                       action={withdrawAndClaimRewards}
+                      preAction={() =>
+                        userDeposit &&
+                        setWithdrawalValue(
+                          transformNumber(
+                            userDeposit,
+                            NumberType.string
+                          ) as string
+                        )
+                      }
```

Comment:
I think as this is now the same code as `max` I'd have pulled out a function, but I also think this is a neat solution so I won't lose sleep!